integration-tests/smoke/vrfv2plus_test.go

```diff
@@ -1364,6 +1367,211 @@ func TestVRFV2PlusWithBHS(t *testing.T) {
+			Msg("Creating and funding subscriptions, adding consumers")
+		fundedSubIDs, err := vrfv2plus.CreateFundSubsAndAddConsumers(
+			env,
+			fundingAmt,
+			fundingAmt,
+			linkToken,
+			vrfv2PlusContracts.CoordinatorV2Plus,
+			[]contracts.VRFv2PlusLoadTestConsumer{vrfv2PlusContracts.VRFV2PlusConsumer[1]},
+			1,
+		)
+		require.NoError(t, err, "error creating funded sub in replay test")
+		randomWordsFulfilledEvent, err := vrfv2plus.RequestRandomnessAndWaitForFulfillment(
+			vrfv2PlusContracts.VRFV2PlusConsumer[1],
+			vrfv2PlusContracts.CoordinatorV2Plus,
+			vrfv2PlusData,
+			fundedSubIDs[0],
+			isNativeBilling,
+			*configCopy.VRFv2Plus.General.MinimumConfirmations,
+			*configCopy.VRFv2Plus.General.CallbackGasLimit,
+			*configCopy.VRFv2Plus.General.NumberOfWords,
+			*configCopy.VRFv2Plus.General.RandomnessRequestCountPerRequest,
+			*configCopy.VRFv2Plus.General.RandomnessRequestCountPerRequestDeviation,
+			configCopy.VRFv2Plus.General.RandomWordsFulfilledEventTimeout.Duration,
+			l,
+		)
+		require.NoError(t, err, "error requesting randomness and waiting for fulfilment")
+		require.False(t, randomWordsFulfilledEvent.OnlyPremium, "RandomWordsFulfilled Event's `OnlyPremium` field should be false")
+		require.Equal(t, isNativeBilling, randomWordsFulfilledEvent.NativePayment, "RandomWordsFulfilled Event's `NativePayment` field should be false")
+		require.True(t, randomWordsFulfilledEvent.Success, "RandomWordsFulfilled Event's `Success` field should be true")
+
+		// 4. wait for the request timeout (1s more) duration
+		time.Sleep(timeout + 1*time.Second)
+
+		// 5. fund sub so that node can fulfill request
+		err = vrfv2plus.FundSubscriptions(
+			env,
+			fundingAmt,
+			fundingAmt,
+			linkToken,
+			vrfv2PlusContracts.CoordinatorV2Plus,
+			[]*big.Int{subID},
+		)
+		require.NoError(t, err, "error funding subs after request timeout")
+
+		// 6. no fulfilment should happen since timeout+1 seconds passed in the job
+		pendingReqExists, err := vrfv2PlusContracts.CoordinatorV2Plus.PendingRequestsExist(testcontext.Get(t), subID)
+		require.NoError(t, err, "error fetching PendingRequestsExist from coordinator")
+		require.True(t, pendingReqExists, "pendingRequest must exist since subID was underfunded till request timeout")
+
+		// 7. remove job and add new job with requestTimeout = 1 hour
+		vrfNode, exists := nodesMap[vrfcommon.VRF]
+		require.True(t, exists, "VRF Node does not exist")
+		resp, err := vrfNode.CLNode.API.DeleteJob(vrfNode.Job.Data.ID)
+		require.NoError(t, err, "error deleting job after timeout")
+		require.Equal(t, resp.StatusCode, 204)
+
+		chainID := env.EVMClient.GetChainID()
+		config.VRFv2Plus.General.VRFJobRequestTimeout = &blockchain.StrDuration{Duration: time.Duration(time.Hour * 1)}
+		g := errgroup.Group{}
+		var wg sync.WaitGroup
+		wg.Add(1)
+		g.Go(func() error {
+			defer wg.Done()
+			err := vrfv2plus.SetupVRFNode(vrfv2PlusContracts, chainID, config.VRFv2Plus.General,
```

Comment:
I still dont agree that we need to restart the node in this test, only create new job spec
Purpose for this test - is to replay requests
We need to restart the node without increasing the timeout only when there is a reorg happening, and it should be a separate test in this ticket

Reply:
Agreed. Completed